datahub-web-react/build.gradle

```diff
@@ -87,6 +87,14 @@ distZip {
   from 'dist'
 }
 
+task copyAssets {
+  dependsOn yarnBuild
+  copy {
+    from 'dist'
+    into '../datahub-frontend/public/'
```

Comment:
Does this public folder always exist? What if not?

Reply:
It's committed into the project I believe, I clean everything under it, but not the folder itself. I think GitHub just doesn't show it because it's not a file